### ISSUE TITLE
Archetype includes prerequisites tag, gives warning

### ISSUE THREAD

**Issue by ocarlsen** [dropwizard/dropwizard#2624]
The dropwizard-archetype version 1.3.8 includes the 'prerequisites' tag:
```
    <prerequisites>
        <maven>3.0.0</maven>
    </prerequisites>
```
This is intended for maven-plugin projects only.  When you run `mvn package` for example, you will see this warning:
```
[WARNING] The project ... uses prerequisites which is only intended for maven-plugin projects but
not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin.
See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
```

**Comment by nickbabcock** [comment]
This will be fixed when dropwizard 2.0 is released. Ref: https://github.com/dropwizard/dropwizard/pull/2320

**Comment by joschi** [comment]
I've merged it into the Dropwizard 1.3.x release branch, too.

The change will be released with Dropwizard 1.3.9.
